simapp/app_config.go

```diff
@@ -130,8 +129,20 @@ var AppConfig = appconfig.Compose(&appv1alpha1.Config{
 			Config: appconfig.WrapAny(&vestingmodulev1.Module{}),
 		},
 		{
-			Name:   banktypes.ModuleName,
-			Config: appconfig.WrapAny(&bankmodulev1.Module{}),
+			Name: banktypes.ModuleName,
+			Config: appconfig.WrapAny(&bankmodulev1.Module{
+				BlockedModuleAccountsOverride: []string{
+					authtypes.FeeCollectorName,
+					distrtypes.ModuleName,
+					minttypes.ModuleName,
+					stakingtypes.BondedPoolName,
+					stakingtypes.NotBondedPoolName,
+					govtypes.ModuleName,
```

Comment:
I am curious about this line, why is it blocked while it is commented below that it isn't.

Reply:
whoops!

good catch

https://github.com/cosmos/cosmos-sdk/pull/13003